cmd/skopeo/proxy.go

```diff
@@ -166,6 +167,11 @@ type proxyHandler struct {
 	activePipes map[uint32]*activePipe
 }
 
+type layersInfo struct {
+  Success string 
```

Comment:
There already is an error reporting mechanism. Why does this need to exist? Especially when it’s not really used.